src/Composer/StudioPlugin.php

```diff
@@ -42,8 +47,8 @@ public static function getSubscribedEvents()
 
     public function dumpAutoload(Event $event)
     {
-        $path = $event->getComposer()->getPackage()->getTargetDir();
-        $studioFile = "{$path}studio.json";
+        $this->targetDir = realpath($event->getComposer()->getPackage()->getTargetDir());
```

Comment:
From my test, `$event->getComposer()->getPackage()->getTargetDir()` always return `NULL`, and by adding `realpath(NULL)` you're getting the root directory of the package.